src/successModal/SuccessModal.tsx

```diff
@@ -46,11 +47,15 @@ class SuccessModal extends Component<SuccessModalProps> {
     const baseClassName = 'kirk-successModal'
     const successContentId = `${baseClassName}-bodyItem-${uuidv4()}`
 
+    // Without hooks it's not that great because you can only have one contextType
+    console.log(this.context)
+    const successModalSize = this.context === MediaSize.SMALL ? ModalSize.SMALL : ModalSize.LARGE
```

Comment:
It's there possible to export MediaSize as a "value"  to map directly to the size desired

something like:
```javascript
const successModalSize = ModalSize[MediaSize]
```

Reply:
Good point, I think it's possible. I removed this for now because I just want to merge the Provider with no components using it. Then we can think of an approach to remove the `size` prop from some component and use context instead